### ISSUE TITLE
Running sample throws WebWorkerTemplatePlugin error

### ISSUE THREAD

**Issue by michaeldzjap** [nnarhinen/react-pdf#55]
I'm trying to build the sample but when executing `npm run build` from /sample I run into a
```
ERROR in ../~/worker-loader!../~/pdfjs-dist/build/pdf.worker.js
Module build failed: Error: Cannot find module 'webpack/lib/webworker/WebWorkerTemplatePlugin'
...
```
error. Any idea what might be going on? I've made sure to first `npm install` from both the root of the project and from /sample. I'm running npm v5.3.0 and node v8.1.4

The same applies to building /test by the way.

**Comment by wojtekmaj** [comment]
Hmm... I think I know what's up. Even though test project uses webpack, it somehow doesn't wanna share it with react-pdf, which requires it for loading the worker. Maybe my assumption was wrong here, or maybe NPM goes crazy when the path to the main component is actually up from test project root folder. I'll investigate, thanks, glad to know it's not only me!

**Comment by wojtekmaj** [comment]
Hey @michaeldzjap, thanks again for reporting that, it's fixed now in 1.8.3 :) Well, at least I hope so!

**Comment by michaeldzjap** [comment]
Thanks for the quick response and fix. I can confirm that I can build and run both /example and /test now.
